custom/scripts/Jobs/send_to_conductor.py

```diff
@@ -221,7 +221,7 @@ def onOKButtonClicked(self):
             self.conductorJob.instance_type = self.selectedInstanceType
             self.conductorJob.instance_count = self.deadlineJob.TaskCount
             self.conductorJob.job_title = self.jobNameTextBox.text()            
-            self.conductorJob.preemptible = self.preemptibleCheckBox.isChecked()       
+            self.conductorJob.preemptible = (self.cloud_provider != "CW" and self.preemptibleCheckBox.isChecked())

```

Comment:
`CW` needs to be a constant.

Reply:
I replaced with a method to improve readability